tools/api_download.py

```diff
@@ -25,6 +25,7 @@ def download_course_instance(args, logfile):
     log(logfile, f'starting download at {local_iso_time()} ...')
     start_time = time.time()
     course_instance_path = f'/course_instances/{args.course_instance_id}'
+    course_instance_info = get_and_save_json(f'{course_instance_path}', 'course_instance_info', args, logfile)
```

Comment:
nit: no need for the f-string, you can use the raw `course_instance_path`.

Reply:
Yep.  The need for that went away after I removed the "course_instance_info" part of the URL per your earlier comment.